cdap-docs/examples-manual/source/examples/spark-page-rank.rst

```diff
@@ -23,8 +22,13 @@ to the *backlinkURLStream*, which stores the URL pair event in its entirety.
 After these events are streamed, they are taken up by the *SparkPageRankProgram*, which
 goes through the entries, calculates page rank and tabulates results in an ObjectStore dataset, *ranks*.
 
+A MapReduce job uses the output of the Spark program from *ranks* dataset and computes the total number of pages for every unique page rank and tabulates its results in another ObjectStore dataset, *rankscount*.
+
+The *PageRankWorkflow* ties the Spark and MapReduce to run together sequentially in this application.
```

Comment:
"to run together sequentially" sounds like an oxymoron. Should it not be "to run sequentially"? (as versus "to run together in parallel")...